Description: Add missing LSB Description to the init script.
Author: Patrick Matthäi <pmatthaei@debian.org>
Reviewed-by: Chris Knadle <chris.knadle@coredump.us>
Origin: https://sourceforge.net/p/mumble/bugs/863/
Last-Update: 2013-11-07

--- a/scripts/murmur.init
+++ b/scripts/murmur.init
@@ -9,6 +9,7 @@
 # Default-Start:        2 3 4 5
 # Default-Stop:         0 1 6
 # Short-Description:    Mumble VoIP Server
+# Description:          init script for the Mumble VoIP Server murmurd.
 ### END INIT INFO
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
