From e8f627f8d52eae97ad8aff6bc24209b1a0e9426c Mon Sep 17 00:00:00 2001
From: Sven-Hendrik Haase <sh@lutzhaase.com>
Date: Tue, 14 Jul 2015 18:47:35 +0200
Subject: [PATCH] Add fallback include path for speech-dispatcher

---
 src/mumble/mumble.pro | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mumble/mumble.pro b/src/mumble/mumble.pro
index 15a4c4f..7d8cbeb 100644
--- a/src/mumble/mumble.pro
+++ b/src/mumble/mumble.pro
@@ -550,6 +550,7 @@ speechd {
 		PKGCONFIG *= speech-dispatcher
 	} else {
 		LIBS *= -lspeechd
+		INCLUDEPATH	*= /usr/include/speech-dispatcher
 	}
 }
 
-- 
2.5.0

